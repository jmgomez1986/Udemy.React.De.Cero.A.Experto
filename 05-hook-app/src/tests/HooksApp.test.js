import { shallow } from 'enzyme';
import { HooksApp } from '../HooksApp';

describe('Pruebas en el componente <HooksApp />', () => {
	test('debe mostrar el componente <HooksApp /> correctamente ', () => {
		const wrapper = shallow(<HooksApp />);

		expect(wrapper).toMatchSnapshot();
	});
	
});
