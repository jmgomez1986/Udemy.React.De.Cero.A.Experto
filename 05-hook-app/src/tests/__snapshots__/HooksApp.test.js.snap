// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el componente <HooksApp /> debe mostrar el componente <HooksApp /> correctamente  1`] = `
<div>
  <h1>
    Hola Mundo!!!
  </h1>
</div>
`;
